src/libraries/System.Drawing.Common/src/System/Drawing/Font.cs

```diff
@@ -291,6 +291,34 @@ public unsafe void ToLogFont(object logFont, Graphics graphics)
             }
         }
 
+        public unsafe void ToLogFont<T>(T logFont, Graphics graphics)
```

Comment:
Aren't you changing the public API surface here? Shouldn't this go through a API design review meeting? System.Drawing.Common ships as a package which doesn't contain a contract assembly and hence customers would start seeing this method.

Reply:
Honestly I'm not much experienced in this process, so decide to put some code. Should I create separate issue for API proposal?

Reply:
Here's the API review process: https://github.com/dotnet/runtime/blob/main/docs/project/api-review-process.md.

Reply:
https://github.com/dotnet/runtime/issues/70358 Submitted